analytics/folder_stats.sql

```diff
@@ -0,0 +1,141 @@
+/* 
+This script contains two parts: 1. lay out folder structure of the targeted FOLDER_IDs;
+2. calculate total data size in GiB for each FOLDER_ID.
+FOLDER_IDs can be a single Synapse folder ID or a list of Synapse folder IDs seperated by comma
+*/
+
+USE ROLE DATA_ANALYTICS;
+USE DATABASE synapse_data_warehouse;
+USE WAREHOUSE COMPUTE_XSMALL;
+
+-- The list of folders to be checked
+SET
+    FOLDER_IDs = '';
```

Comment:
This is not working for me. I tried a folder with ID `syn55226438`. In this field, I tried:
`syn55226438` and `55226438`.

In both cases I am getting this error back:
```
error line 0 at position -1
'SYS_VW.PARENT_ID_5' in select clause is neither an aggregate nor in the group by clause.
```

It seems like it is pointing to this statement:
```
SELECT
    'syn' || nodesnapshots.FOLDER_ID AS FOLDER_ID,
    'syn' || nodesnapshots.PARENT_ID AS PARENT_ID,
    'syn' || nodesnapshots.ID AS ID,
    COUNT(nodesnapshots.ID) AS COUNTS,
    nodesnapshots.NAME,
    nodesnapshots.NODE_TYPE,
    nodesnapshots.FILE_HANDLE_ID,
    filesnapshots.CONTENT_SIZE
FROM
    nodesnapshots
JOIN 
    synapse_data_warehouse.synapse.file_latest AS filesnapshots 
ON nodesnapshots.file_handle_id = filesnapshots.ID
GROUP BY nodesnapshots.FOLDER_ID;
```

Reply:
Thanks a lot for catching this. I have to look at it in depth. I don't remember seeing this error before. Maybe I copied the wrong script.